OpenRA.Mods.Common/Traits/MovementSounds.cs

```diff
@@ -0,0 +1,65 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using OpenRA.Support;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	public class MovementSoundsInfo : ITraitInfo
+	{
+		[Desc("Randomly select a sound from this list to play when this actor beings moving.")]
+		public readonly string[] Sounds = { };
+
+		[Desc("Should the selected sound loop?")]
+		public readonly bool LoopSound = true;
```

Comment:
You believe, I confirm. RA2 sound.ini had a property called loop for that which wasn't default at all.